src/NotifyIconWpf/Interop/WindowMessageSink.cs

```diff
@@ -242,8 +248,11 @@ private void ProcessWindowMessage(uint msg, IntPtr wParam, IntPtr lParam)
             switch (message)
             {
                 case WindowsMessages.WM_CONTEXTMENU:
-                    // TODO: Handle WM_CONTEXTMENU, see https://docs.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shell_notifyiconw
-                    Debug.WriteLine("Unhandled WM_CONTEXTMENU");
+                    ContextMenuReceived?.Invoke(new Point()
```

Comment:
As mentioned in the other comment, why not place the code to do the low / high bit logic directly here, as it's probably not used elsewhere.